client/app/styles/onboarding.styl

```diff
@@ -33,13 +33,24 @@ header
             display: block
 
         svg
-            max-width: 100%
-            height:    (200 / 16)em
```

Comment:
@m4dz : I had to reduce size, but I didnt understand this ratio; could you explain  it to me so that I can fix this?